### DISCUSSION TITLE
Nose wheel steering

### DISCUSSION THREAD

**Discussion by 1ancaster** [flybywiresim/aircraft#4484] [Help] [answered]
When using the EFB pushback the NW steering is disconnected and shown in amber. However, it is not possible to reactivate it as expected. When the msfas pushback or Push Helper are used this does not occur. Can this please be addressed?

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
It is disconnected still as the logic assumes that the pushback tug is still connected to the nose wheel even though it has driven away. You need to press the call tug button again to re-enable nw strg.
